addons/sale/i18n/sale.pot

```diff
@@ -4,10 +4,9 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 17.0\n"
```

Comment:
I think there has been a mistake here, I guess I first remove the line because it had a `+e` at the end and I might have removed the whole line in the end. I can reopen a PR on my side to fix the line if needed.

Reply:
Please do so (and fix it in the forward-port of this PR since you already have a conflict anyway).
I'm not aware of the real implications of this line in the .pot files, but it could be problematic if it blocked the synchronization from Transifex.